src/ui/Analytics/PendingSearchEvent.ts

```diff
@@ -140,5 +140,11 @@ export class PendingSearchEvent {
         searchEvent.customData[`context_${key}`] = value;
       });
     }
+
+    // The refinedKeywords field is important for Reveal in order to learn properly on query
+    // made based on the big fuzzy query.    
+    if (queryResults.refinedKeywords != undefined && queryResults.refinedKeywords.length != 0) {
+      searchEvent.customData['refinedKeywords'] = queryResults.refinedKeywords;
```

Comment:
Have you discussed this parameter name with the UA team ?

For example, for the context data (that you can see on line 140), we mutually agreed on a special format, so that the UA service can ignore and treat that custom dimension in a special manner (eg: you can't do dashboard on that dimension, meaning it's much less stressful on their infrastructure)